tasks/omero-install.yml

```diff
@@ -169,3 +169,15 @@
   notify:
     - omero-server rewrite omero-server configuration
     - omero-server restart omero-server
+
+- name: omero server | patch bin/omero to use virtualenv
+  become: true
+  replace:
+    mode: 0755
+    path: "{{ omero_server_basedir }}/{{ omero_server_symlink }}/bin/omero"
+    regexp: "^#!.+python$"
+    replace: "#!{{ omero_server_basedir }}/venv/bin/python"
```

Comment:
:+1:

Reply:
:+1: we might want to implement something similar in the `ome.omero_web` role